zebra-consensus/src/transaction.rs

```diff
@@ -539,14 +601,14 @@ where
 
         Self::verify_v5_transaction_network_upgrade(&transaction, upgrade)?;
 
-        let shielded_sighash = transaction.sighash(upgrade, HashType::ALL, None);
+        let shielded_sighash =
+            transaction.sighash(upgrade, HashType::ALL, Default::default(), None);
```

Comment:
This seems inconsistent with v4, can you document why it's different?

Or apply this fix:
```suggestion
            transaction.sighash(upgrade, HashType::ALL, cached_ffi_transaction.all_previous_outputs().clone(), None);
```

Reply:
Good catch!